model/Software/Classes/PackageHash.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# PackageHash
```

Comment:
This looks *very* much like /Core/Hash -- I mean, it even has the exact same properties. Why is it here?

If there is really need for a Hash as a verification, there should be a Class with a single property, of type `/Core/Hash`.

Reply:
Yes, that is what I was saying. Get rid of PackageHash and just have Hash

Reply:
`PackageHash` is a subclass of `PackageVerification` whereas `Hash` is not.

We are using the type restrictions to restrict what a `PackageVerification` can be.  If we use this approach, then we need a separate class.

If we want to take a different approach, someone needs to describe it in a pull request or some other form of documentation - at this point, I don't see any other reasonable alternatives.

@maxhbr @zvr - It sounds like you don't agree with this approach - can you provide specifics on an alternative that provides a method for verifying packages?  See issue #345 for the alternatives I came up with.  Also note that this PR is the result of a discussion in the tech call where we agreed to go down this path.